addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/DeviceHandler.java

```diff
@@ -209,27 +213,27 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             case CHANNEL_RADIATOR_MODE:
                 if (command instanceof StringType) {
                     if (command.equals(MODE_ON)) {
-                        state.getHkr().setTist(HeatingModel.TEMP_FRITZ_ON);
+                        state.getHkr().setTsoll(HeatingModel.TEMP_FRITZ_ON);
```

Comment:
This seems to make sense, but was it just wrong before?

Reply:
Yes it was. E.g. for `CHANNEL_SETTEMP` and an `INCREASE` / `DECREASE` command, we read and increase / decrease the value of `Tsoll`. The new value should be assigned to `Tsoll` again after we manipulated it.

`Tist` is the measured temperature and `Tsoll` is the expected temperature.